game-app/game-core/src/main/java/games/strategy/triplea/delegate/UserActionDelegate.java

```diff
@@ -259,13 +262,18 @@ private void notifySuccess(final UserActionAttachment userActionAttachment) {
     bridge
         .getSoundChannelBroadcaster()
         .playSoundForAll(SoundPath.CLIP_USER_ACTION_SUCCESSFUL, player);
-    final UserActionText uat = UserActionText.getInstance(bridge.getResourceLoader());
-    final String text = userActionAttachment.getText();
-    sendNotification(uat.getNotificationSuccess(text));
-    notifyOtherPlayers(
-        userActionAttachment,
-        uat.getNotificationSuccessOthers(text),
-        uat.getNotificationSuccessTarget(text));
+    bridge
+        .getResourceLoader()
+        .ifPresent(
+            resourceLoader -> {
```

Comment:
Nit: Up to you, but this construct is a bit verbose.

This would be more concise:
```java
ResourceLoader resourceLoader = bridge.getResourceLoader().orElse(null);
if (resourceLoader != null) {
  ...
}
```